dns/zone.py

```diff
@@ -37,19 +37,19 @@
     from io import StringIO
 
 class BadZone(dns.exception.DNSException):
-    """The zone is malformed."""
+    """The DNS zone is malformed."""
     pass
 
 class NoSOA(BadZone):
-    """The zone has no SOA RR at its origin."""
+    """The DNS zone has no SOA RR at its origin."""
     pass
 
 class NoNS(BadZone):
-    """The zone has no NS RRset at its origin."""
+    """The DNS zone has no NS RRset at its origin."""
     pass
 
 class UnknownOrigin(BadZone):
-    """The zone's origin is unknown."""
+    """The DNS zone's origin is unknown."""
     pass
 
 class Zone(object):
```

Comment:
btw, thanks for this changeset. In my application I am currently catching misc. of these exceptions and creating my own error msgs for them. So you likely helped to simplify my code at some time in the future. \o/
the python3 port of dnspython needs similar, but not same changes btw.